plugins/browser-plugin-media/src/schemata.ts

```diff
@@ -0,0 +1,79 @@
+import { MediaPlayerEventType } from './types';
+
+export const MEDIA_PLAYER_SCHEMA = 'iglu:com.snowplowanalytics.snowplow/media_player/jsonschema/1-0-0';
+export const MEDIA_PLAYER_SESSION_SCHEMA = 'iglu:com.snowplowanalytics.snowplow/media_player_session/jsonschema/1-0-0';
+export const MEDIA_PLAYER_AD_SCHEMA = 'iglu:com.snowplowanalytics.snowplow/media_player_ad/jsonschema/1-0-0';
+export const MEDIA_PLAYER_AD_BREAK_SCHEMA =
+  'iglu:com.snowplowanalytics.snowplow/media_player_ad_break/jsonschema/1-0-0';
+export const MEDIA_PLAYER_QUALITY_SCHEMA = 'iglu:com.snowplowanalytics.snowplow/media_player_quality/jsonschema/1-0-0';
+export const MEDIA_PLAYER_ERROR_SCHEMA = 'iglu:com.snowplowanalytics.snowplow/media_player_error/jsonschema/1-0-0';
+
+function eventNameForEventType(eventType: MediaPlayerEventType): string {
+  switch (eventType) {
```

Comment:
I think we can generate a dict at runtime from the enum instead of this switch, to save a bit of space from strings. I did a similar thing in one of the media trackers, though it requires a bit of type finagling. You should be able to do something like:

```
type MediaPlayerEventTypeKeyMap = {
  [key in MediaPlayerEventType]: string;
};

// Allows us to create an empty object of type
let mediaPlayerEventTypeKeyMap = {} as MediaPlayerEventTypeKeyMap;

// Fill the empty object with key/values and add the prefix to the values
const prefix = "media_player_event_";
Object.keys(MediaPlayerEventType).forEach((k) => {
  mediaPlayerEventTypeKeyMap[MediaPlayerEventType[k as keyof typeof MediaPlayerEventType]]
    = prefix + MediaPlayerEventType[k as keyof typeof MediaPlayerEventType]
});


console.log(mediaPlayerEventTypeKeyMap[MediaPlayerEventType.Ready]); // "media_player_event_ready" 
```

Reply:
Good point, it really was too verbose! I couldn't do the 1:1 translation of the event types to schema names because a couple of types have different names (ad first quartile, midpoint, third quartile). So I left a switch for them, but otherwise just decided to concat strings with the event type.

Reply:
Yeah that's much tider than my suggestion too! Perhaps a small comment to explain why the switch is still required, otherwise looks good!